src/code42cli/cmds/cases.py

```diff
@@ -241,6 +243,10 @@ def add(state, case_number, event_id):
     """Associate a file event to a case, by event ID."""
     try:
         state.sdk.cases.file_events.add(case_number, event_id)
+    except Py42UpdateClosedCaseError:
+        raise
+    except Py42CaseAlreadyHasEventError:
+        raise
     except Py42BadRequestError:
         raise Code42CLIError("Invalid case-number or event-id.")
 
```

Comment:
There are other spots in this file where we are doing error handling that I would expect could be removed now that we have added the custom exception to the global catcher

Reply:
This is being discussed [here](https://github.com/code42/code42cli/pull/222#discussion_r569296895), resolving this comment to avoid multiple discussion on the same topic.